api/qsharp/toc.yml

```diff
@@ -1,3 +1,5 @@
+- uid: microsoft.quantum.qsharplibintro
```

Comment:
This TOC.yml file is automatically generated and will be overwritten by our current publication pipeline.

Reply:
Thanks, @cgranade.  Is the entire qsharp/api/qsharp folder overwritten or are the files just updated? I ask because we need to keep an index.md file in that folder to prevent a 404 error. I can workaround not using the toc.yml file.

Reply:
Reverted the changes on api/qsharp/toc.yml

Reply:
> Thanks, @cgranade. Is the entire qsharp/api/qsharp folder overwritten or are the files just updated? I ask because we need to keep an index.md file in that folder to prevent a 404 error. I can workaround not using the toc.yml file.

The process is fairly manual at this point, in that we copy the files emitted by the build by hand into qsharp/api/qsharp. Those files include toc.yml such that changes there are definitely overwritten, but changes to index.md can be kept if we're very careful when updating API docs.